### PR TITLE
acrn-config: enable tpm for hybird_rt on ehl

### PR DESCRIPTION
**Pull request by szhen11**
enable tpm for hybrid_rt scenario on ehl board. And generate configuration source code based on latest config.

Tracked-On: #5506

Signed-off-by: Shuang Zheng <shuang.zheng@intel.com>

### CODE REVIEW

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by wenlingz** [comment]
@terryzouhao please help approve this PR, thanks

**Review by terryzouhao — APPROVED** [review]

File: misc/acrn-config/library/board_cfg_lib.py [hunk 1]
```diff
@@ -34,7 +34,7 @@
 TSN_DEVS = ["8086:4b30", "8086:4b31", "8086:4b32", "8086:4ba0", "8086:4ba1", "8086:4ba2",
             "8086:4bb0", "8086:4bb1", "8086:4bb2", "8086:a0ac", "8086:43ac", "8086:43a2"]
 GPIO_DEVS = ["8086:4b88", "8086:4b89"]
-TPM_PASSTHRU_BOARD = ['whl-ipc-i5', 'whl-ipc-i7', 'tgl-rvp']
+TPM_PASSTHRU_BOARD = ['whl-ipc-i5', 'whl-ipc-i7', 'tgl-rvp', 'ehl-crb-b']
```
Comment (gvancuts) [inline misc/acrn-config/library/board_cfg_lib.py:37]: Does this mean that if someone wants to create their own board with the `acrn-config`, and wants to expose the TPM2 device on it (as a pass-through device), they **must** also edit this file to make this happen?

File: misc/vm_configs/scenarios/hybrid_rt/ehl-crb-b/misc_cfg.h [hunk 2]
```diff
@@ -76,12 +76,17 @@
 #define VM3_VCPU_CLOS			{0U}
 #endif
 
+#define VM0_PASSTHROUGH_TPM
```
Comment (gvancuts) [inline misc/vm_configs/scenarios/hybrid_rt/ehl-crb-b/misc_cfg.h:79]: This section would also be automatically generated by the `acrn-config` tool if someone were to re-generate the code files from the source XMLs, is that correct?

File: misc/acrn-config/library/board_cfg_lib.py (hunk 1 above)
Comment (szhen11) [reply misc/acrn-config/library/board_cfg_lib.py:37]: Currently we officially supports these boards for TPM passthru and hard coded them here.  If users want their own board configured with TPM passthru to the pre-launched VM, they should also edit this file and configure vm->mmio_resources->TPM2 = y in scenario xml.

File: misc/vm_configs/scenarios/hybrid_rt/ehl-crb-b/misc_cfg.h (hunk 2 above)
Comment (szhen11) [reply misc/vm_configs/scenarios/hybrid_rt/ehl-crb-b/misc_cfg.h:79]: Yes. We should make the xml configuration and configuration source code aligned when updating xmls or config tool.

File: misc/acrn-config/library/board_cfg_lib.py (hunk 1 above)
Comment (gvancuts) [reply misc/acrn-config/library/board_cfg_lib.py:37]: @NanlinXie , this is an example of something we should avoid in our ACRN configuration tool.

**Pull request merged**